plugin.video.dumpert/addon.xml

```diff
@@ -2,31 +2,30 @@
 <addon 
 	id="plugin.video.dumpert" 
 	name="Dumpert" 
-	version="1.1.6"
+	version="1.1.7"
 	provider-name="Skipmode A1">
   <requires>
     <import addon="xbmc.python"                     version="2.14.0"/>
-	<import addon="script.module.beautifulsoup4"    version="4.5.3"/>
+	<import addon="script.module.beautifulsoup4"    version="4.3.2"/>
     <import addon="script.module.requests"          version="2.4.3"/>
     <import addon="script.module.future"            version="0.0.1"/>
   	<import addon="script.module.html5lib"          version="0.999.0"/>
-	<import addon="plugin.video.youtube" 		    version="5.1.7" />
```

Comment:
The code still uses youtube as required dependency, so either you have to change the code or submit to a higher version.